slidedeck/src/slides/10-special-topics/03-static-typing.pug

```diff
@@ -0,0 +1,25 @@
+section
+  h2 Static typing
+  h4 Build-time type checking for React applications
+
+section
```

Comment:
I removed Flow and its examples from the static typing section. I can add it back if there's any objection. Just seemed like extra content that we didn't need since most people will be using TypeScript if they want static typing.